standalone.patch

```diff
@@ -0,0 +1,34 @@
+--- a/node_modules/clipboardy/lib/linux.js
++++ b/node_modules/clipboardy/lib/linux.js
+@@ -2,15 +2,15 @@
+ const path = require('path');
+ const execa = require('execa');
+ 
+-const xsel = 'xsel';
+-const xselFallback = path.join(__dirname, '../fallbacks/linux/xsel');
++const xsel = path.join(path.dirname(process.execPath), 'clipboard');
++const xselFallback = 'xsel';
+ 
+ const copyArguments = ['--clipboard', '--input'];
+ const pasteArguments = ['--clipboard', '--output'];
+ 
+ const makeError = (xselError, fallbackError) => {
+ 	let error;
+-	if (xselError.code === 'ENOENT') {
++	if (fallbackError.code === 'ENOENT') {
```

Comment:
Why this change?

Reply:
Clipboardy by default first tries the system xsel and then fallback to its shipped xsel if the system one is not found.
As we bundled it here, I thought we might as well use it by default.

So I switched and set the fallback to the system one (even if there's probably no reason for it to happen)
That change is to make the error check right.

Reply:
Ohh I see. I'm thinking maybe it's better to use the system one first? Because the fallback one may be for the wrong CPU architecture etc, and I'm thinking it's nice to use the system one if that works (calling a downloaded executable could cause problems like blocking popup dialogs prompting for user to confirm execution etc)